AGENTS.md

```
@@ -1,60 +1,223 @@
+# AGENTS.md
+
+## Project Overview
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)